### ISSUE TITLE
In `gins_server.go`,the `server.GetName()` is always equal `default`.

### ISSUE THREAD

**Issue by Cheene** [gogf/gf#1811]
### 1. What version of `Go` and system type/arch are you using?

go version go1.16.6 darwin/amd64

### 2. What version of `GoFrame` are you using?

github.com/gogf/gf/v2 v2.0.4

### 3. Can this issue be re-produced with the latest release?

Yes

### 4. What did you do?

I wanted to open a two-port service, but it never worked.
**profile**
```
server:
  address: ":8080"
  server.http:
    address: ":8081"
```
**start**
```
Func: func(ctx context.Context, parser *gcmd.Parser) (err error) {
			s := g.Server("http")
			s.Run()
			return nil
		},
```
### 5. What did you expect to see?
```
http server started listening on [:8080] 
```

### 6. What did you see instead?
```
http server started listening on [:8081] 
```

### 7. Why?
In the file `github.com/gogf/gf/v2@v2.0.4/frame/gins/gins_server.go`
```
serverConfigMap = Config().MustGet(
				ctx,
				fmt.Sprintf(`%s.%s`, configNodeName, server.GetName()),
			).Map()
```
The `server.GetName()` is always equal `default`.

Reason:

Step 1.
In the file `github.com/gogf/gf/v2@v2.0.4/frame/gins/gins_server.go`
```
return localInstances.GetOrSetFuncLock(instanceKey, func() interface{} {
		server := ghttp.GetServer(instanceName)
      ....
```

Step 2. go to `ghttp.GetServer(instanceName)`
In the file `github.com/gogf/gf/v2@v2.0.4/net/ghttp/ghttp_server.go`
```
// Initialize the server using default configurations.
if err := s.SetConfig(NewConfig()); err != nil {
	panic(gerror.WrapCode(gcode.CodeInvalidConfiguration, err, ""))
}
```

Step 3. go to `NewConfig()`
In the file `github.com/gogf/gf/v2@v2.0.4/net/ghttp/ghttp_server_config.go`
```
return ServerConfig{
		Name:                DefaultServerName,
		....
```
Step 4. go to `server.GetName()`
In the file `github.com/gogf/gf/v2@v2.0.4/net/ghttp/ghttp_server_config.go`
```
func (s *Server) GetName() string {
	return s.config.Name
}
```
So, the `server.GetName()` is always equal `default`,not `http`.

**Comment by greenhandatsjtu** [comment]
> In the file `github.com/gogf/gf/v2@v2.0.4/frame/gins/gins_server.go`
> 
> ```
> serverConfigMap = Config().MustGet(
> 				ctx,
> 				fmt.Sprintf(`%s.%s`, configNodeName, server.GetName()),
> 			).Map()
> ```
This has been fixed in v2.0.5, you can use latest version.
https://github.com/gogf/gf/blob/v2.0.5/frame/gins/gins_server.go#L62-L65 : 
![image](https://user-images.githubusercontent.com/40566803/167826156-01113d2e-4b2f-457d-ace0-6d5846bd00d0.png)

Besides, refer to the [document](https://goframe.org/pages/viewpage.action?pageId=44449484), you should replace `server.http` with `http` like below:
```yaml
server:
  address: ":8080"
  http:
    address: ":8081"
```
Now, it works well:
![image](https://user-images.githubusercontent.com/40566803/167826753-cd9a3860-7bcd-4fa2-97de-7f7e7dbf4abc.png)

**Comment by Cheene** [comment]
Thanks.
